enos/modules/fetch_artifactory/variables.tf

```diff
@@ -28,7 +28,7 @@ variable "artifactory_repo" {
 
 variable "edition" {
   type        = string
-  description = "The edition of the binary to search, it can be either CE or ENT"
+  description = "The edition of the binary to search, it can be either \"ce\" or \"ent\""
```

Comment:
stuff like this can be enforced with variable validation:
https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules

Reply:
Done in https://github.com/hashicorp/nomad/pull/25051/commits/9fe4e5747036ab2541d18c8d6bba627dbfa14825